dags/common/utils.py

```diff
@@ -11,10 +12,17 @@ def set_harvesting_interval(repo, **kwargs):
             "start_date": kwargs["params"]["start_date"],
             "until_date": kwargs["params"]["until_date"],
         }
-    start_date = repo.find_the_last_uploaded_file_date()
-    until_date = date.today()
-
+    start_date = datetime.datetime.strptime(
+        repo.find_the_last_uploaded_file_date(), "%Y-%m-%d"
```

Comment:
you are already formatting the date in `find_the_last_uploaded_file_date`

Reply:
` start_date = datetime.datetime.strptime(
        repo.find_the_last_uploaded_file_date(), "%Y-%m-%d"
    ).date() ` 0 is a type date
    
    while:
`"start_date": (start_date or until_date).strftime("%Y-%m-%d"),
        "until_date": until_date.strftime("%Y-%m-%d"),` - I'm stringify it

Reply:
we don't need all these date transformations, let's stick to one. Preferably always use `strftime` since we always need str date for our functions.